include/secp256k1_ecdsa_adaptor.h

```diff
@@ -10,7 +10,24 @@ extern "C" {
  *  Lloyd Fournier
  *  (https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-November/002316.html
  *  and https://github.com/LLFourn/one-time-VES/blob/master/main.pdf).
-*/
+ *
+ *  WARNING! DANGER AHEAD!
+ *  As mentioned in Lloyd Fournier's paper, the adaptor signature leaks the
+ *  Diffie-Hellmann key between the signing key and the encryption key. This is
```

Comment:
```suggestion
 *  Diffie-Hellman key between the signing key and the encryption key. This is
```

Reply:
Mabye also mention the acronym ECDH here to make sure everyone gets this (because that's the name of the corresponding library module).